app/main.ipynb

```diff
@@ -1,312 +1,469 @@
+     },
+     "execution_count": 43,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "adult = pd.read_csv(\"../data/adult.csv\", index_col=[0]).replace(\"?\", np.nan)\n",
+    "adult.dropna(inplace=True)\n",
+    "adult"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Task 1\n",
+    "Print the count of men and women in the dataset."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "(21790, 10771)"

```

Comment:
Strange here. You have a dataframe with 30162 rows
But here you have 21790 + 10771 rows, which more than input

Reply:
I forgot update all task after added data cleaning in 0 task(